rzk/grammar/Syntax.cf

```diff
@@ -106,6 +110,7 @@ RecOrDeprecated.  Term7 ::= "recOR" "(" Term "," Term "," Term "," Term ")" ;
 -- Types
 TypeFun.        Term1 ::= ParamDecl "→" Term1 ;
 TypeSigma.      Term1 ::= "Σ" "(" Pattern ":" Term ")" "," Term1 ;
+TypeSigmaNested. Term1 ::= "Σ" "(" [SigmaParam] ")" "," Term1 ;
```

Comment:
I think this rule overlaps with the previous one.

Reply:
Indeed, should probably be `SigmaParam "," [SigmaParam]`